### ISSUE TITLE
Value are not correct

### ISSUE THREAD

**Issue by FreezyEx** [dappuniversity/price-bot#10]
If you check the values on uniswap you can see they are not the same of the program

**Comment by RarebatDeveloper** [comment]
Yes, the value on uniswap is not accurate. Whereas for kyberswap, it seems accurate

**Comment by stonebirdarchitect** [comment]
I assume no fixes came this way ?
